Packs/MicrosoftEntraID/README.md

```diff
@@ -16,10 +16,41 @@
 - RiskyServicePrincipals
 - ServicePrincipalRiskEvents
 
-#### Supported Timestamp Formats:
+### Supported Timestamp Formats:
 - YYYY-MM-DDTHH:MM:SS.S* (UTC)
 - YYYY-MM-DDTH:M:S.S* (UTC)
 
+***
+
+## Data Collection
+To configure Microsoft Entra ID to send logs to Cortex XSIAM, follow the below steps.
+
+### Prerequisites
+- Create an **Azure event hub**. For more information, refer to Microsoft's official [documentation](https://learn.microsoft.com/en-us/azure/event-hubs/event-hubs-create).
+- Make sure that you have at least Security Administrator role.
+
+### Stream logs to an event hub
+1. Sign in to the **Microsoft Entra admin center**.
+2. Navigate to **Identity** &rarr; **Monitoring & health** &rarr; **Diagnostic settings**.
+3. Select **+ Add diagnostic setting** to create a new integration or select **Edit setting** for an existing integration.
```

Comment:
Capitalize if the options are capitalized in the official page

Reply:
Not capitalized in the official page